cmd/buildah/images.go

```diff
@@ -195,14 +198,14 @@ func parseFilter(ctx context.Context, store storage.Store, images []storage.Imag
 		case "before":
 			beforeDate, err := setFilterDate(ctx, store, images, pair[1])
 			if err != nil {
-				return nil, errors.Errorf("no such id: %s", pair[0])
+				return nil, errors.Errorf("invalid filter: no such id: %s", pair[0])
```

Comment:
Since we passed `pair[1]` to `setFilterDate`, the error message should probably be embedding the value of `pair[1]` instead of `pair[0]`.  Also applies to the "since" case.  We're also discarding `err`, but I don't have an easy solution for that.

Reply:
You always get punished for attempting to clean up code that we not covered in your original PR.

Reply:
Fixed and now I wrap the err.